.ci.yaml

```diff
@@ -366,11 +366,6 @@ targets:
       version_file: flutter_master.version
       target_file: android_build_all_packages.yaml
       channel: master
-      # The legacy project build requires an older JDK.
```

Comment:
Unfortunately it looks like the tool still supports a version of Gradle old enough to require 11, so it's no longer possible to test the existing legacy project and the current template on the same setup.

Unless we want to lose some legacy coverage, I think we'll need to split out a new task, making regular build-all and legacy build-all entirely separate (with a TODO to recombine them once we drop the version of Gradle/AGP/etc. that requires 11). @reidbaker do you see any other option?

Reply:
Yep, I was looking at the `legacy` version of the app that is [now failing](https://ci.chromium.org/ui/p/flutter/builders/try/Linux_android%20android_build_all_packages%20master/9871/overview).

Reply:
That was essentially the motivation for setting up a Java 11 specific shard in https://github.com/flutter/flutter/pull/151433. Sorry, I should have thought about how this would affect the packages legacy test.

Reply:
@gmackall I think you did exactly what @stuartmorgan is saying, splitting the legacy vs non-legacy builds. I'll prepare something similar for flutter/packages!